sanic/pages/styles/BasePage.css

```diff
@@ -1,28 +1,57 @@
 /** BasePage **/
+
+:root {
+    --sanic: #ff0d68;
+    --sanic-blue: #0369a1;
+    --sanic-yellow: #ffde41;
+    --sanic-purple: #7e22ce;
+    --sanic-green: #4d7c0f;
+    --sanic-background: #f1f5f9;
+    --sanic-text: #1f2937;
+    --sanic-tab-background: #fff;
+    --sanic-tab-text: #0f172a;
+    --sanic-tab-shadow: #adadad;
+    --sanic-highlight-background: var(--sanic-yellow);
+    --sanic-highlight-text: var(--sanic-text);
+}
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --sanic: #ef4444;
```

Comment:
this value is meant to be Sanic's on brand color. It should not differ in dark mode.